math/maxima/Portfile

```diff
@@ -19,30 +18,62 @@ long_description \
         tensors. Maxima yields high precision numeric results \
         by using exact fractions, arbitrary precision integers, \
         and variable precision floating point numbers. Maxima \
-        can plot functions and data in two and three dimensions.
+        can plot functions and data in two and three dimensions. \n\
+        Maxima includes the following commands: \n\
+        * \"rmaxima\" is a front end which provides advanced line-editing \
+        facilities via rlwrap. \n\
+        * \"xmaxima\" is a graphical interface for maxima, written in Tcl/Tk.
 
 homepage        http://maxima.sourceforge.net/
-master_sites    sourceforge
 
-checksums       md5     3bccafb512ff1054be700c61601088af \
-                sha256  f58ff98013fcaa37d8661b941672c38db58f982f082d6cfcca1f1b5fac7d71b3 \
-                rmd160  3c4b153554c9b9cb3442e030d503fbe975a614fc
+subport maxima-devel {
+    conflicts   maxima
+
+    # git describe --tags : branch-5_41-base-415-g7e71fc4
+    # Date:  Fri Jun 15 17:26:53 2018 -0500
+    # commit 7e71fc4b6a95cab6418227754f01fc44ac845da1
+    version     5.41post
+    fetch.type  git
+    git.url     https://git.code.sf.net/p/maxima/code
+    git.branch  7e71fc4b6a95cab6418227754f01fc44ac845da1
+
+    pre-configure {
+        system -W ${worksrcpath} "./bootstrap"
```

Comment:
https://sourceforge.net/p/maxima/code/ci/master/tree/bootstrap, right? I think `autoreconf` can also generate these files, and `use_autoreconf` can add the required dependencies (autoconf, automake, libtool), without which CI would fail.

```
NAME
       autoreconf - Update generated configuration files

SYNOPSIS
       autoreconf [OPTION]... [DIRECTORY]...

DESCRIPTION
       Run  `autoconf'	(and  `autoheader', `aclocal', `automake', `autopoint'
       (formerly `gettextize'), and `glibtoolize' where  appropriate)  repeat-
       edly  to remake the GNU Build System files in specified DIRECTORIES and
       their subdirectories (defaulting to `.').
```

Reply:
> $ cat ./bootstrap 
> #!/bin/sh
> set -x
> aclocal
> automake --add-missing --copy
> autoconf

and
https://sourceforge.net/p/maxima/code/ci/master/tree/INSTALL says the following:

>  UNIX GNU AUTOTOOLS BUILD AND INSTALL
>  ====================================
>  
>  IF CONFIGURE SCRIPT IS ABSENT
>  -----------------------------
>  
>  Execute ``sh bootstrap'' to create the Maxima configure script.
>  (configure is absent if the source code is obtained directly
>  from GIT instead of rpm or tar.gz release file.)

But if 'use_autoreconf yes' has the same effect, I will 'use_autoreconf'.

Reply:
As long as it builds with `use_autoreconf yes` (and removing this pre-configure block), I'm sure `autoreconf` has the same effect.